content/src/main/content/META-INF/vault/filter.xml

```diff
@@ -62,6 +62,7 @@
         <include pattern="/etc/acs-commons/audit-log-search(.*)?" />
     </filter>
     <filter root="/apps/cq/core/content/nav/tools/acs-commons"/>
+    <filter root="/apps/wcm/core/content/editor/jcr:content/content/items/content/header/items/headerbar/items/url"/>
```

Comment:
We shouldn't add this button by default, rather provide via documentation (and a supporting package/XML snippet) what needs to be included.